File: Projects/rosalind/IEV.java
import java.io.*;
import java.util.*;

public class IEV {
	public static void main(String[] args) throws IOException {
		Scanner in = new Scanner(System.in);
		double ans = 0;
		ans += 2 * (in.nextInt() + in.nextInt() + in.nextInt());
		ans += 1.5 * in.nextInt() + in.nextInt();
		System.out.println(ans);
		in.close();
	}
}

File: Projects/rosalind/IEV.kt
package rosalind

import java.io.IOException
import java.util.*

object IEV {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        var ans = 0.0
        ans += 2 * (stdin.nextInt() + stdin.nextInt() + stdin.nextInt()).toDouble()
        ans += 1.5 * stdin.nextInt() + stdin.nextInt()
        println(ans)
        stdin.close()
    }
}
